prusa/connect/printer/cameras.py

```diff
@@ -242,6 +286,30 @@ def camera_id(self):
         """Returns this camera's ID"""
         return self._driver.camera_id
 
+    @property
+    def fingerprint(self):
+        """Returns the camera ID as a fingerprint for connect"""
+        return make_fingerprint(self.camera_id)
+
+    @property
+    def is_registered(self):
+        """Is the camera registered to connect?"""
+        return self._token is not None
+
+    @property
+    def token(self):
+        """Return the camera token"""
+        return self._token
```

Comment:
It we use token directly from another class, simple use token attribute instead of property. If we want to do some magic on setting proces... do it really private and define setter.